src/cdk/testing/tests/protractor.e2e.spec.ts

```diff
@@ -392,11 +391,16 @@ describe('ProtractorHarnessEnvironment', () => {
     });
 
     it('should throw when multiple queries fail to match', async () => {
-      await expectAsyncError(() => harness.missingElementsAndHarnesses(),
-          'Error: Failed to find element matching one of the following queries:' +
-          '\n(TestElement for element matching selector: ".not-found"),' +
-          '\n(SubComponentHarness with host element matching selector: "test-sub" satisfying' +
-          ' the constraints: title = /not found/)');
+      try {
+        await harness.missingElementsAndHarnesses();
+        fail('Expected to throw.');
+      } catch (e) {
+        expect(e.message).toBe(
```

Comment:
can't we use `expectAsync(...).toBeRejectedWithError(...)`?

Reply:
Not in protractor tests since Protractor v5 doesn't support jasmine `3.x`, so `expectAsync` is not even available even though our types currently suggest it (also without changes from this PR).

We can't update to v6 since that is not supported. See our discussion on Slack (for later reference):
https://angular-team.slack.com/archives/G0HSN89A4/p1582057111144900

Reply:
ah gotcha